syntax/yang.vim

```diff
@@ -85,8 +85,8 @@ hi def link yangConditional Conditional
 hi def link yangType Type
 
 " arbitrary mappings
-hi def link yangKeyword Type
-hi def link yangStatement Type
```

Comment:
This seems reasonable and seems to look okay on my Let's move these two up into the previous "things with one-to-one mapping" section since they do, in fact, seem to have direct mappings.